### ISSUE TITLE
Link to "how to contribute" page and add a nudge to submit PRs to fix stale issues

### ISSUE THREAD

**Issue by jlengstorf** [gatsbyjs/peril-gatsbyjs#83]
@rally25rs had a great idea in https://github.com/yarnpkg/yarn/issues/7110#issuecomment-473467670 to include a nudge to contribute on older issues.

If we add a sentence or two that keeps Gatsbot's friendly tone and lets people know that the best way to see an issue fixed is to open a PR + link to [gatsby.dev/contribute](https://gatsby.dev/contribute), it may help encourage more contributors (and certainly won't hurt in any case).

**Comment by JDeLay-JR** [comment]
@jlengstorf I'd love to take this issue if it's still open!

**Comment by jlengstorf** [comment]
@JDeLay-JR that would be great — thank you!

The message that needs editing is here: https://github.com/gatsbyjs/peril-gatsbyjs/blob/master/tasks/stale.ts

Thanks again!

**Comment by JDeLay-JR** [comment]
@jlengstorf For sure! I also recently opened and closed PR for gastbyjs/peril for a different message regarding Gatsby swag and subsequently opened a new one. I apologize for the duplicate PR, the first was made using my old GitHub account that was still connected to my GitHub desktop application on my PC. I have fixed that issue and submitted a new PR so I just wanted to give you (and the other maintainers) a heads up!
